tests/bluetooth/bsim/audio/src/tbs_client_test.c

```diff
@@ -22,8 +23,14 @@ static volatile bool call_placed;
 static volatile uint8_t call_state;
 static volatile uint8_t call_index;
 static volatile uint8_t tbs_count;
+static volatile bool call_terminated;
 
+CREATE_FLAG(provider_name);
 CREATE_FLAG(ccid_read_flag);
+CREATE_FLAG(signal_str);
```

Comment:
consider 'signal_strength' (full name - to be consistent with the others)

Reply:
Done